coach/src/user/menu.cjsx

```diff
@@ -38,7 +38,7 @@ UserMenu = React.createClass
     @context.navigator.emit('show.student_id', view: 'student_id')
 
   update: ->
-    @forceUpdate() if @isMounted()
+    @forceUpdate()
```

Comment:
I'm guessing this component never unmounts, since it's the menu? Nothing in here calls this `update` function, but not sure if something external could call it.